raspiKernelInfo.sh

```diff
@@ -53,32 +76,51 @@ function extractStageDescription() {
 	fi
 }
 
-if [[ $1 == "-f" ]] ; then
-    displayAndExec "tail -4 /proc/cpuinfo"
+COMMANDS=""
```

Comment:
You use these variables as flags for true and false. Your code works perfectly. I prefer to initialize them with 0 or 1 for false and true and use `(( ))` for any tests. That way you can use readable logical operators `&&` and `||` inside of `(( ))` to combine and logical variables.